modules/administration/pages/master-fingerprint.adoc

```diff
@@ -1,11 +1,21 @@
 [[security-master-fingerprint]]
 = Set up a Client to Master Validation Fingerprint
 
-In highly secure network configurations you may wish to ensure your Salt clients are connecting a specific master.
-To set up validation from client to master enter the master's fingerprint within the [path]``/etc/salt/minion`` configuration file, see the following procedure:
 
-.Procedure: Adding Master's Fingerprint to Client
+In highly secure network configurations you may wish to ensure your {salt} clients are connecting a specific master.
+To set up validation from client to master start by entering the master's fingerprint within the configuration file:
+
+* [path]``/etc/salt/minion.d/custom.conf`` in cases of using classic {salt} minion in your client, or
+* [path]``/etc/venv-salt-minion/minion.d/custom.conf`` in case of using {salt} Bundle in your client
```

Comment:
These mentioned `custom.conf` files are not existing by default on a system, the user should have to create.

So I would rephrase to:

```suggestion
In highly secure network configurations you may wish to ensure your {salt} clients are connecting a specific master.
To set up validation from client to master start by entering the master's fingerprint within a {salt} minion configuration file:

* [path]``/etc/salt/minion.d/custom.conf`` in cases of using classic {salt} minion in your client, or
* [path]``/etc/venv-salt-minion/minion.d/custom.conf`` in case of using {salt} Bundle in your client
```